wiki/Tournaments/ATS/2018/en.md

```diff
@@ -472,18 +495,14 @@ The Asian Taiko Showdown 2018 is run by various osu!taiko community members.
 ### Scheduling Instructions
 
 1. Each stage will be held on **a single weekend**.
-
 2. All Double Elimination Stages will be held on either Saturday or Sunday, UTC +7.
-
 3. Scheduling will be handled by the Tournament Management. Schedules will be released on the Sunday after current stage is finished. Reschedules are allowed, however must be agreed on by both players and proof must be provided.
```

Comment:
> Tournament Management

This position does not exist (or another name for `Organisers`). It should be consistent, but idk the term to use.